python/ray/rllib/pg/pg.py

```diff
@@ -0,0 +1,79 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import numpy as np
+
+import ray
+from ray.rllib.optimizers import LocalSyncOptimizer
+from ray.rllib.pg.pg_evaluator import PGEvaluator, RemotePGEvaluator
+from ray.rllib.agent import Agent
+from ray.tune.result import TrainingResult
+
+DEFAULT_CONFIG = {
+    # Number of workers (excluding master)
+    "num_workers": 4,
+    # Size of rollout batch
+    "batch_size": 512,
+    # Discount factor of MDP
+    "gamma": 0.99,
+    # Learning rate
+    "lr": 0.0001,
+    # Model and preprocessor options
+    "model": {},
+    # Arguments to pass to the rllib optimizer
+    "optimizer": {
+        # Number of gradients applied for each `train` step
+        "grads_per_step": 10,
+    },
+    "model_options": {},
+    # Arguments to pass to the env creator
+    "env_config": {},
+}
+
+
+class PGAgent(Agent):
```

Comment:
"""Simple policy gradient agent.

This is an example agent to show how to implement algorithms in RLlib. In most cases, you will probably want
to use the PPO agent instead."""

Reply:
Could you also switch the link in http://ray.readthedocs.io/en/latest/rllib-dev.html from A3CEvaluator to PGAgent instead? (rllib-dev.rst file under doc/source)

Reply:
Done; I switched the link to be to PGEvaluator instead of PGAgent though (since it's supposed to be an example of an evaluator class).